### ISSUE TITLE
Write a contributor communication SOP

### ISSUE THREAD

**Issue by lauraclarke** [ebi-ait/hca-ebi-wrangler-central#67]
# As a wrangler I want to understand how  and when to follow up with contributors during the submission process

This goal of this task is to define an SOP about follow up with contributors after an initial expression of interest in contributing data to the HCA or after curation of a publication

The questions about this process are

- How to best support non-responsive contributors?
- How to improve the rate of response?
- How frequently to repeat questions when they don't respond
- What is a reasonable escalation policy when a lab member isn't responded to more senior lab members or even funders
- Can we improve how we write our emails to get responses?

This SOP should be linked to the general new contributor SOP https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/contributor_dataset_SOP.md

## acceptance criteria

- [ ] An contributor follow up SOP accepted by all wranglers in https://github.com/ebi-ait/hca-ebi-wrangler-central/tree/master/docs/SOPs

This SOP will need to be reviewed and success metrics to be considered by how frequently people respond in the first instance and for non-responders, how frequently we need to escalate

We, of course, need to consider which parts of this are under our control and which parts aren't

**Comment by lauraclarke** [comment]
@mshadbolt @rays22 @clairerye I don't know if there is other things I have missed here

**Comment by rays22** [comment]
> @mshadbolt @rays22 @clairerye I don't know if there is other things I have missed here

I think the questions in the task description look quite complete. I could not think of any major point to add, maybe only a minor one.
I think it is implied in the goals and acceptance criteria, but to make it explicit for the SOP, I think we also need to think about how to track dates of communications/responses. This would be useful for both the success metrics and for the wranglers to know when it is time to nudge a contributor again.

**Comment by mshadbolt** [comment]
I have started a draft here: https://docs.google.com/document/d/1XC3p8TdgDiJGVuJXYIJppWkBsKWKOhw_fhPLC5A0W3g/edit

Not really ready for a full review yet but if people want to add anything or suggest any changes as it currently is feel free

**Comment by mshadbolt** [comment]
I received feedback on the draft from Claire, Laura and wranglers. 

There may be some minor changes to some of the template emails but I decided it was 'done enough' to put it on to the wrangler documentation website here: https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/contributor_communication_SOP.html

I also made some links to the new page from the contributor wrangling SOP. 

I also added links to the contributor communication SOP to the visual index.
